File: sykepengesoknad/src/main/java/no/nav/syfo/kafka/sykepengesoknad/dto/FravarDTO.java
package no.nav.syfo.kafka.sykepengesoknad.dto;

import lombok.Builder;
import lombok.Value;

import java.time.LocalDate;

@Value
@Builder(toBuilder = true)
public class FravarDTO {
    LocalDate fom;
    LocalDate tom;
    FravarstypeDTO type;
}

File: src/main/kotlin/no/nav/helse/flex/sykepengesoknad/kafka/FravarDTO.kt
package no.nav.helse.flex.sykepengesoknad.kafka

import java.time.LocalDate

data class FravarDTO(
    val fom: LocalDate? = null,
    val tom: LocalDate? = null,
    val type: no.nav.helse.flex.sykepengesoknad.kafka.FravarstypeDTO? = null
)
